gaia-header.js

```diff
@@ -295,20 +317,26 @@ module.exports = component.register('gaia-header', {
    * Run font-fit on a title with
    * the given amount of content space.
    *
-   * @param  {String} text
-   * @param  {Number} space
+   * @param  {Object} {space, text, [min], [max] }
    * @return {Object} {fontSize, textWidth}
    * @private
    */
-  fontFit: function(text, space) {
-    debug('font fit', text, space);
-    return fontFit({
+  fontFit: function(opts) {
+    debug('font fit:', opts);
+
+    // these are the default values.
+    var fontFitArgs = {
       font: TITLE_FONT,
-      space: space,
-      text: text,
-      min: 16,
-      max: 24
-    });
+      min: MINIMUM_FONT_SIZE_UNCENTERED,
+      max: MAXIMUM_FONT_SIZE
+    };
+
+    // but the options can override them.
+    for (var key in opts) {
+      fontFitArgs[key] = opts[key];
+    }
```

Comment:
It's a little confusing to me if this isn't a requirement right now. AFAICT `text` and `space` are 'required' and `min` is 'optional', for this reason perhaps the arguments should should look like:

```js
fontfit: function(text, space, options) {

}

// ...

this.fontFit(text, space, { min: ... });
```